libs/accounts/src/lib/accounts-manager.tsx

```diff
@@ -29,6 +31,7 @@ export const AccountManager = ({
   isReadOnly,
   pinnedAsset,
   noBottomPlaceholder,
+  gridActionsDropdownTrigger,
```

Comment:
@MadalinaRaicu as discussed lets
- remove this prop
- move the dropdown trigger to libs/datagrid
- render the dropdown trigger directly where its needed

Reply:
while doing the actual changes I can see it's cleaner to:

- [ ] render the dropdown trigger directly  
- [ ] remove the prop

I think, you're right. The component is **very** small, it's not worth it to make the grid actions trigger as its own component anymore.

```html
                      <DropdownMenu
                        trigger={
                          <DropdownMenuTrigger
                            iconName="more"
                            className="hover:bg-vega-light-200 
                            dark:hover:bg-vega-dark-200 p-0.5 focus:rounded-full hover:rounded-full"
                          ></DropdownMenuTrigger>
                        }
                      >
```